test/unit/gateways/go_cardless_test.rb

```diff
@@ -53,6 +53,21 @@ def test_appropriate_purchase_amount
     assert_equal 1000, response.params['payments']['amount']
   end
 
+  def test_successful_refund
+    @gateway.expects(:ssl_request)
+       .with(:post, 'https://api-sandbox.gocardless.com/refunds', anything, anything)
```

Comment:
thats a really cool succinct way to write that out.  I am gonna start using this!